src/Entity/ClientUser.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * (c) Adrien PIERRARD
+ */
+
+namespace App\Entity;
+
+use App\Repository\ClientUserRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Entity Class ClientUser.
+ *
+ * @ORM\Entity(repositoryClass=ClientUserRepository::class)
+ * @UniqueEntity(fields={"email"}, message="This ClientUser already exists", groups={"user"})
+ */
+class ClientUser
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var Client
+     *
+     * @ORM\ManyToOne(targetEntity=Client::class, inversedBy="clientUsers")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $client;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=50)
+     *
+     * @Assert\NotBlank(message="You must add a name", groups={"user"})
+     * @Assert\Length(
+     *     min=3,
+     *     max=50,
+     *     minMessage="The name should contain at least {{ limit }} characters",
+     *     maxMessage="The name should not contain more than {{ limit }} characters",
+     *     allowEmptyString=false,
+     *     groups={"user"}
+     * )
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=30, unique=true)
+     *
+     * @Assert\NotBlank(message="You must add a username", groups={"user"})
```

Comment:
Validation group will not be used 
```suggestion
     * @Assert\NotBlank(message="You must add a username")
```